index.html

```diff
@@ -5,157 +5,190 @@
-                        <li class="item6">Ruby</li>
-                    </ul>
-                </div>
-                <div class="container4">
-                    <button class="button1">See Projects</button>
-                </div>
-                <div class="container2">
-                    <div class="Data"><h1>Profesional Art Printing Data</h1></div>
-                    <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
-                    <ul class="container3">
-                        <li class="item4">css</li>
-                        <li class="item5">bootstrap</li>
-                        <li class="item6">Ruby</li>
-                    </ul>
+                    <div class="listo">
+                        <ul class="a">
+                            <li class="item1">css</li>
+                            <li class="item2">html</li>
+                            <li class="item3">bootstrap</li>  
+                            <li class="item2">Ruby</li>  
+                        </ul>
+                        <button class="button">See Projects</button>
+                    </div>
                 </div>
-                <div class="container4">
-                    <button class="button1">See Projects</button>
+                <div class="containo">
+                        <div class="container2">
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
+                                </ul>
+                            </div>
+                                <div class="container4">
+                                    <button class="button1">See Projects</button>
+                                </div>
+                        </div>
+                        <div class="container2">
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
```

Comment:
[OPTIONAL]  I suggest that you fix how the grid behaves from `768px` to ensure a consistent design on all breakpoints.
[see here](https://duanedave.github.io/Portfolio-Website/) play with the responsiveness and see.